client/src/components/Button.js

```diff
@@ -23,6 +24,7 @@ const Button = styled.button`
 
   p {
     margin-left: 20px;
+    font-size: 16px;
```

Comment:
Maybe set this font-size in GlobalStyle if it is your standard size?

Reply:
Yes, your are right! THX 👍🏻